### PR TITLE
sass loader dropped

### PR DESCRIPTION
**Pull request by arturkulig**
removes sass-loader
removes SCSS sheets and CSS sheets in general

cld-image styles are moved to inline styles as they should not be overridden

### CODE REVIEW

File: dist/Cloudinary.common.js [hunk 1]
```diff
@@ -1877,23 +1783,38 @@ function noop() {}
     },
     imageAttrs: function imageAttrs() {
       var className = {
-        "cld-image": true,
-        "cld-fill": this.responsive === "fill",
-        "cld-fill-width": this.responsive === "width",
-        "cld-fill-height": this.responsive === "height"
+        "cld-image": true
       };
+      var responsiveStyle = {
```
Comment (mayashavin) [inline dist/Cloudinary.common.js]: do we really need to apply our style here? if so, why?

File: dist/demo.html [hunk 2]
```diff
@@ -1,10 +1,7 @@
...
+<meta charset="utf-8" />
 <title>Cloudinary demo</title>
 <script src="./Cloudinary.umd.js"></script>
 
-<link rel="stylesheet" href="./Cloudinary.css">
-
-
 <script>
-console.log(Cloudinary)
+  console.log(Cloudinary);
```
Comment (mayashavin) [inline dist/demo.html]: remove console?

File: src/components/CldImage.vue [hunk 3]
```diff
@@ -110,11 +110,27 @@ export default {
     },
     imageAttrs() {
       const className = {
-        "cld-image": true,
-        "cld-fill": this.responsive === "fill",
-        "cld-fill-width": this.responsive === "width",
-        "cld-fill-height": this.responsive === "height"
+        "cld-image": true
       };
+      const responsiveMode = this.responsive === "" ? "width" : this.responsive;
```
Comment (mayashavin) [inline src/components/CldImage.vue]: again, why do we need inline style?

File: dist/Cloudinary.common.js (hunk 1 above)
Comment (arturkulig) [reply dist/Cloudinary.common.js]: These styles were always here, they are now as inline styles.
They are inlined as there is no reason for letting the user to override them.

The rules are for stretching an image to its parent width/height/size when "responsive" behaviour is enabled. This way the proper size is always maintained (user should set a size of a parent-container, not to the image element)  and if an image size changes and image file with a different size is necessary - it is then loaded.

File: dist/demo.html (hunk 2 above)
Comment (arturkulig) [reply dist/demo.html]: Please notice that this console.log is in a particular file - an HTML file that can be opened locally, for demoing purposes and so console.log here is for signifying that the `Cloudinary` object is available.

**Pull request merged**